src/tribol/geom/GeomUtilities.cpp

```diff
@@ -1099,6 +1099,7 @@ bool SegmentIntersection2D( const real xA1, const real yA1, const real xB1, cons
    }
 
    // TODO refine how these debug calculations are guarded
+#ifdef AXOM_DEBUG
```

Comment:
When/how is this defined now? Does this properly address issue https://github.com/LLNL/Tribol/issues/39?